code/ai/aiturret.cpp

```diff
@@ -573,6 +573,8 @@ void evaluate_obj_as_target(object *objp, eval_enemy_obj_struct *eeo)
 			}
 		}
 
+		Assertion(Objects[eeo->turret_parent_objnum].type == OBJ_SHIP,"evaluate_obj_as_target was about to call object_is_targetable with a non-ship object.  Please report!");
```

Comment:
This is impossible, the assertion condition is the same as the if branch this is contained in. If we wanted to make sure `object_is_targetable()` is only run on ships, then that function should probably assert this, although it seems to be able to handle multiple different object types.

Reply:
You are correct it was redundant.  I think it's supposed to be able to handle more than one type of object, so I'm not going to make any changes there.